ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/ConfigManagementServiceIT.scala

```diff
@@ -71,6 +71,10 @@ final class ConfigManagementServiceIT extends LedgerTestSuite {
           newTimeModel = oldTimeModel,
         )
         .mustFail("setting a time model with an expired MRT")
+
+      // Above operation finished on a timeout, but may still succeed asynchronously.
+      // Stabilize the ledger state before leaving the test case.
+      _ <- stabilize(ledger)
```

Comment:
but here we still have to read after stabilizing, and making sure that what we read is not the failed one (as that one will fail anyway as we discussed)
or is this what you wanted to have a separate PR for?

Reply:
I also don't understand why we wait for the participant to catch up, but don't read from it afterwards.

Reply:
@mziolekda Do you want to move this back to draft?

Reply:
This is making sure that the unsettled state of the ledger does not spill over to the subsequent test. Without stabilizing the subsequent test may pick-up noise and fail for all the wrong reasons.

Reply:
Oh I see where the confusion comes from, allocateParty is a write+read operation. It waits until party allocation arrives in teh index-db.

Reply:
> making sure that the unsettled state of the ledger does not spill over to the subsequent test

I see. `setTimeModel` is _also_ a read+write operation, blocking until the change was witnessed in the index db. However, the last call to `setTimeModel` is using an already expired MRT, which means the participant will immediately abort waiting for the confirmation of the change.

You also just swap the "Restore the original time model" and "Try to set a time model with an expired MRT." blocks. The "Restore the original time model" call will serve the same purpose as the `stabilize` call.